client/src/components/Main/RSVPCard/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+
+import Card from "../../Common/Card";
+
+import { borderRadius } from "../../../resources/style-constants";
+
+const CardContainer = styled(Card)`
+  height: 75px;
+  width: 390px;
+  border-radius: ${borderRadius};
```

Comment:
If you are just changing the height and width of the Card, you probably don't need to declare the rest of the css properties. Try commenting out the following properties and see if the component looks correct.